### PR TITLE
Add incubator/vaultingkube chart

### PR DESCRIPTION
**Pull request by jody-frankowski**
Add a chart for [vaultingkube](https://github.com/sunshinekitty/vaultingkube).

Ping @sunshinekitty

### CODE REVIEW

**Comment by unguiculus** [comment]
Excellent. Thank you.

File: incubator/vaultingkube/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
```
Comment (unguiculus) [inline incubator/vaultingkube/templates/deployment.yaml]: apps/v1beta2

Comment (jody-frankowski) [reply incubator/vaultingkube/templates/deployment.yaml]: Unless I'm mistaken, I understood that this repository aims to support the latest stable version of k8s, and I understood from the reference documentation that the api version of `Deployment` is `v1` https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#deployment-v1-apps
Could you please explain why i'm wrong :)

Comment (jody-frankowski) [reply incubator/vaultingkube/templates/deployment.yaml]: As a side note, I have only seen the idea of supporting the latest stable version of k8s api in an issue and not in any docs (the docs only say `Do not leverage alpha features of Kubernetes` which does not precise which stable versions of k8s should be supported).

Comment (unguiculus) [reply incubator/vaultingkube/templates/deployment.yaml]: We official support the current and previous minor versions. CI is not at 1.9 yet either.

See https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md#compatibility

Comment (jody-frankowski) [reply incubator/vaultingkube/templates/deployment.yaml]: Done :)

**Comment by timstoop** [comment]
@jody-frankowski Could you take a look at the errors produced by the e2e tests, please?

**Comment by jody-frankowski** [comment]
@timstoop Well the chart is expected to fail if the user doesn't provide a certain number of values. While this chart could depend on the Vault chart and configure a Vault if needed, I don't really think it's a good idea, and I don't think too many people would expect this chart to do this.
I tried to follow [this contribution guideline](https://github.com/kubernetes/charts/blame/master/CONTRIBUTING.md#L37)

**Comment by unguiculus** [comment]
@jody-frankowski The chart can't be merged if it doesn't pass CI. You may provide special values files for testing if that helps.

See https://github.com/kubernetes/charts/blob/master/test/README.md#providing-custom-test-values

**Comment by jody-frankowski** [comment]
@unguiculus  I reproduced what the [minecrat chart did](https://github.com/kubernetes/charts/blob/3bc3476d81095ab4c59a037770586bfb773e13ba/stable/minecraft/templates/deployment.yaml#L1) because until the ci lets me use another chart for the tests, I won't be able to produce a `Running` pod even with the tests ci values.

**Comment by sunshinekitty** [comment]
Thanks @jody-frankowski - I'll get documentation on vaultingkube updated on using this chart.

**Pull request merged**